vnext/Microsoft.ReactNative.Cxx/Crash.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#define VerifyElseCrash(condition) \
+  do {                             \
+    if (!(condition)) {            \
+      assert(false && #condition); \
+      std::terminate();            \
+    }                              \
+  } while (false)
+
+#define VerifyElseCrashSz(condition, message) \
```

Comment:
>Sz [](start = 23, length = 2)

Not a fan of adding something with Hungrian notation to RNW, even if it matches what we have in devmain.

Reply:
Any proposal for a different name?

Reply:
VerifyElseCrashMessage?

We also might have something in Folly that does this that might be worth reusing.

---
In reply to: [356831337](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356831337) [](ancestors = 356831337)

Reply:
Would be nice to grab it from Open Source Mso :)

---
In reply to: [356901187](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356901187) [](ancestors = 356901187,356831337)

Reply:
Yes, but I just needed something to crash conditionally for now. 

---
In reply to: [356912397](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356912397) [](ancestors = 356912397,356901187,356831337)